docs/shared/monorepo-ci-github-actions.md

```diff
@@ -17,31 +14,29 @@ jobs:
   main:
     runs-on: ubuntu-latest
     steps:
-      - uses: actions/checkout@v3
+      - uses: actions/checkout@v4
         with:
           fetch-depth: 0
-      - uses: nrwl/nx-set-shas@v3
+      - uses: actions/setup-node@v3
+        with:
+          node-version: 20
+          cache: 'npm'
```

Comment:
What is the reason for reordering these? (not that it makes any difference)

Reply:
Swapping the order of `npm ci` and `nx-set-shas` was not intentional.  I moved some lines around to get the npm cache step to work and probably swapped those lines in that process